src/components/Icons/Edit.vue

```diff
@@ -1,13 +1,26 @@
 <template>
   <svg
```

Comment:
Is there a reason there's no height/width for some of these? I just went and copied the svg in the figma for this icon and theres a height and width of 24 each?

Reply:
I can't remember, if we were supposed to delete the height/width or add it? but in any case shouldn't they all be with or all without it?